### DISCUSSION TITLE
clicking on task opens new tab.

### DISCUSSION THREAD

**Discussion by MarkosMuche** [community/community#16328] [Projects and Issues] [answered]
Priorly, we could edit the tasks and their descriptions by pressing on the task. But now when you press the tab, it opens a new tab and loads the same page. Why is that?

**Comment by egost** [comment]
I am having the exact same problem. This is true for me when I click on a Draft issue in the Board layout.
https://www.loom.com/share/27384d8508e34b5c92a21c6d479d2310

**Comment by egost** [comment]
Duplicate here: https://github.com/github/feedback/discussions/16164

**Comment by mariorod** [comment, marked as answer]
This should now be fixed. Answering as such. If it is not please let us know.
